boards/msb-430/include/periph_conf.h

```diff
@@ -21,19 +21,28 @@
 #ifndef PERIPH_CONF_H
 #define PERIPH_CONF_H
 
+#include "periph_cpu.h"
+#include "macros/units.h"
+
 #ifdef __cplusplus
 extern "C" {
 #endif
 
-/**
- * @name    Clock configuration
- * @{
- */
-/** @todo   Move all clock configuration code here from the board.h */
 #define CLOCK_CORECLOCK     (7372800U)
```

Comment:
How did this ever work? The clock was initialized to run at ~2.5 MHz from DCO due to

```C
#define MSP430_INITIAL_CPU_SPEED    2457600uL
#define F_CPU                       MSP430_INITIAL_CPU_SPEED
#define F_RC_OSCILLATOR             32768
#define MSP430_HAS_DCOR             1
#define MSP430_HAS_EXTERNAL_CRYSTAL 0
```

But `CLOCK_CORECLOCK` was ~7.4 MHz ans so was `CLOCK_CMCLK`, which was used to configure the UART symbol rate to 9600 Baud. The actual symbol rate like was then 3200 Baud due to this mismatch :-/

Reply:
The good thing is that the compile time configuration sanity tests do indeed work and caught the issue.